File: Java/src/main/java/org/sisuns/bucles_for/basico/Ejercicio_05.java
package org.sisuns.bucles_for.basico;

public class Ejercicio_05 {
    public static void main(String[] args) {
        int N = 5; // Puedes cambiar este valor según sea necesario
        int suma = 0;
        int contador = 0;
        int numero = 2;

        while (true) {
            boolean es_primo = true;

            for (int i = 2; i <= Math.sqrt(numero); i++) {
                if (numero % i == 0) {
                    es_primo = false;
                    break; // Salir del bucle interno si no es primo
                }
            }

            if (es_primo) {
                suma += numero;
                contador++;

                if (contador == N) {
                    break; // Salir del bucle principal si se han encontrado N números primos
                }
            }

            numero++;
        }

        // Mostrar la suma de los primeros N números primos
        System.out.println("La suma de los primeros " + N + " números primos es: " + suma);
    }
}

File: PHP/ESTRUCTURA_BUCLES/FOR/Basico/Ejercicio_05.php
<?php
    $N = 5;
    $suma = 0;
    $contador = 0;
    for ($numero = 2; $numero <= $N * 20; $numero++) {
        //Este tipo de variable es conocida como booleana.
        $es_primo = true;

        for ($i = 2; $i <= sqrt($numero); $i++) {
            if ($numero % $i == 0) {
                $es_primo = false;
                break;
            }
        }

        if ($es_primo) {
            $suma += $numero;
            $contador++; 
        }
        if ($contador == $N) {
            break;
        }
    }

    echo "La suma de los primeros ".$N." números primos es: ".$suma;
?>